oaknational/Oak-Web-Application
# PUPIL-532 demo(lottie-file): test lottie file on app

## Description

Music year: 2012

- List of changes
display lottie file on owa pages

acceptance criteria

- [x]  Tested locally on a dedicated page
- [x]  Tested remotely via unmerged PR
- [x]  Tested with public assets and with cloudinary storage
- [x]  Tested looping / non-looping
- [x]  Tested rendering at different sizes

## Issue(s)

Fixes #

## How to test

1. Go to [mascot](https://deploy-preview-2314--oak-web-application.netlify.thenational.academy/pupils/test-animade)
2. you should see clear animated mascott

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
